signals.mml

```diff
@@ -98,8 +98,8 @@ Layer:
             NULL AS construction_usage, NULL AS construction_service,
             NULL AS preserved_railway, NULL AS preserved_service,
             NULL AS preserved_usage,
-            pzb, lzb, atb, atb_eg, atb_ng, atb_vv, etcs, construction_etcs,
-            railway_train_protection_rank(pzb, lzb, atb, atb_eg, atb_ng, atb_vv, etcs, construction_etcs) AS rank
+            pzb, lzb, atb, atb_eg, atb_ng, atb_vv, kvb, tvm, etcs, construction_etcs,
+            railway_train_protection_rank(pzb, lzb, atb, atb_eg, atb_ng, atb_vv, kvb, tvm, etcs, construction_etcs) AS rank
```

Comment:
If you add new arguments to the `railway_train_protection_rank` function, you have to add support for those in `sql/functions.sql` as well.

Reply:
OK, I have changed `sql/functions.sql`
Can you have a look?
Thanks :)